client/scripts/views/modals/create_community_modal.ts

```diff
@@ -1,173 +1,186 @@
-import 'modals/create_community_modal.scss';
+// import 'modals/create_community_modal.scss';
+import 'modals/manage_community_modal.scss';
 
 import m from 'mithril';
 import $ from 'jquery';
-import _ from 'lodash';
-import mixpanel from 'mixpanel-browser';
-import { Button } from 'construct-ui';
-
 import app from 'state';
+import mixpanel from 'mixpanel-browser';
+import { Table, Button } from 'construct-ui';
 
-// import User from 'views/components/widgets/user';
 import { CompactModalExitButton } from 'views/modal';
-import { CommunityInfo } from 'models';
+import { notifyError } from 'controllers/app/notifications';
+import { InputPropertyRow, TogglePropertyRow } from './manage_community_modal/metadata_rows';
+
 
 interface IAttrs {}
 
 interface IState {
   disabled: boolean;
   error: string;
   success: string | boolean;
-  selectedAddress: string;
-  selectedChain: string;
+  name: string;
+  description: string;
+  iconUrl: string;
+  invitesEnabled: boolean;
+  privacyEnabled: boolean;
+  isAuthenticatedForum: boolean;
+  website: string;
+  discord: string;
+  element: string;
+  telegram: string;
+  github: string;
+  customDomain: string;
```

Comment:
how is domain different from website?

Reply:
My mistake, it was present in manage community modal, and I copied it here too